Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -489,6 +491,67 @@ def split_fields(fields: str = '', delimiter: str = ';') -> dict:
     return dic_fields
 
 
+def split_notes(raw_notes, note_type, time_info):
+    notes: List = []
+    notes_split = raw_notes.split('\n\n')
+    retrieved_last_note = False
+    for note in notes_split:
+        if not note:
+            continue
+        if 'Mirrored from Cortex XSOAR' in note:
+            if retrieved_last_note:  # add to last note only in case the note was not filtered by the time filter
+                notes[-1]['value'] += '\n\n Mirrored from Cortex XSOAR'
+            continue
+        note_info, note_value = note.split('\n')
+        created_on, created_by = note_info.split(' - ')
+        created_by = created_by.split(' (')[0]
+
+        # convert note creation time to UTC
+        created_on_UTC = datetime.strptime(created_on, DATE_FORMAT) + time_info.get('timezone_offset')
+
+        if time_info.get('filter') and created_on_UTC < time_info.get('filter'):
+            # If a time_filter was passed and the note was created before this time, do not return it.
+            demisto.debug(f'Using time filter: {time_info.get("filter")}. Not including note: {note}.')
+            retrieved_last_note = False
+            continue
+        note_dict = {
+            "sys_created_on": created_on_UTC.strftime(DATE_FORMAT),
+            "value": note_value,
+            "sys_created_by": created_by,
+            "element": note_type
+        }
+        notes.append(note_dict)
+        retrieved_last_note = True
```

Comment:
what if 2 notes were added?

Reply:
This boolean was added to handle the case of a note created in XSOAR (as I explained [here](https://github.com/demisto/content/pull/22509#discussion_r1046698364) about the structure of notes). It shouldn't be a problem even if two notes are added as we handle them in the for loop here and the '\n\n Mirrored from Cortex XSOAR' message should always be added to the previous note that was retrieved.